contracts/CourtRoom.sol

```diff
@@ -37,7 +45,6 @@ contract CaseContract is Owned {
 		return clm.claimId;
 	}
```

Comment:
@MrTibbles Giving that another thought , we do need the OpenCases to handle multiple clients in parallel.

If the client is compensated - its claim is resolve and  it should automatically "leaveGame" by the contract. (It should be added). So it can not open a newClaim again unless it is re register by the service. So the only way it can compensated twice is when the service register it again .
The service can assure that it is not register a client more than one time on the same block so this guaranty that each claim will get the a unique id(sha3(_plaintiff, _evidence, now) ).

If the client case is seems not valid (and not compensated) - its still register to game (unless it removed by a registration timeout expiration) and can open a new case , which can be just overrun the current case if it has the same id (due submission on the same block).

The event log can be use to track the history of claim upon each contract decision..
Does it make sense ?